exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/project/ProjectRecordBatch.java

```diff
@@ -324,6 +324,11 @@ protected boolean setupNewSchema() throws SchemaChangeException {
               if (name == EMPTY_STRING) {
                 continue;
               }
+
```

Comment:
This has a major documentation impact since there is a change in the DIR columns returned in a star query. Please make a note of this in the JIRA.